assets/src/blocks/ENForm/ENFormInPlaceEdit.js

```diff
@@ -193,34 +204,38 @@ const Signup = ({attributes, setAttributes}) => {
         <div className="title-and-description">
           <RichText
             tagName="h2"
-            value={title}
-            onChange={(title) => { setAttributes({title}) }}
-            placeholder={__('Enter form title', 'planet4-blocks-backend')}
+            value={ title }
+            onChange={ ( titleVal ) => {
+              setAttributes( { titleVal } );
+            } }
+            placeholder={ __( 'Enter form title', 'planet4-blocks-backend' ) }
             withoutInteractiveFormatting
-            allowedFormats={[]}
+            allowedFormats={ [] }
             multiline="false"
           />
           <RichText
             tagName="div"
-            value={description}
+            value={ description }
             className="form-description"
-            onChange={(description) => { setAttributes({description}) }}
-            placeholder={__('Enter form description', 'planet4-blocks-backend')}
-            allowedFormats={['core/bold', 'core/italic']}
+            onChange={ ( descriptionVal ) => {
+              setAttributes( { descriptionVal } );
```

Comment:
Same here, it should be `setAttributes( { description: descriptionVal } );`

Reply:
Done.